ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next__4.6.yaml

```diff
@@ -9,14 +9,6 @@ build_root:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 canonical_go_repository: knative.dev/serving
 images:
-- dockerfile_path: openshift/ci-operator/knative-images/webhook/Dockerfile
```

Comment:
Why do we need to touch 4.6?

Reply:
Same with the comment above. The auto generated script changed the order :disappointed:

Reply:
ah ok